samplekotlin/src/main/java/com/example/chattutorial/ChannelActivity4.kt

```diff
@@ -86,11 +87,12 @@ class ChannelActivity4 : AppCompatActivity() {
         ChatClient
             .instance()
             .channel(cid)
-            .subscribeFor(this, TypingStartEvent::class, TypingStopEvent::class
+            .subscribeFor(this, TypingStartEvent::class.java, TypingStopEvent::class.java
             ) { event ->
                 when (event) {
                     is TypingStartEvent -> currentlyTyping.add(event.user.name)
                     is TypingStopEvent -> currentlyTyping.remove(event.user.name)
+                    else -> Unit
```

Comment:
If this is to remove a warning about an implicit cast to `Any`, better to suppress the warning with a comment

Reply:
It's because `'when' expression on sealed classes is recommended to be exhaustive`, so I don't know what's worse - leave the warning or add else branch even if we know that we won't get any other event 🤔

Reply:
We can suppress it with `@Suppress("NON_EXHAUSTIVE_WHEN_ON_SEALED_CLASS")`, because we are intentionally not adding more branches